php/class-photonfill.php

```diff
@@ -1034,19 +1034,16 @@ public function get_attachment_picture( $attachment_id, $size = 'full', $attr )
 		 */
 		private function build_attachment_image( $attachment_id = null, $attr ) {
 			$attr = apply_filters( 'photonfill_img_attributes', $attr, $attachment_id );
-			if ( ! empty( $attachment_id ) && is_numeric( $attachment_id ) ) {
-				$attachment = get_post( $attachment_id );
 
-				if ( empty( $attr['alt'] ) ) {
-					$attr['alt'] = trim( strip_tags( get_post_meta( $attachment_id, '_wp_attachment_image_alt', true ) ) );
-				}
-				if ( empty( $attr['alt'] ) ) {
-					$attr['alt'] = trim( strip_tags( $attachment->post_excerpt ) ); // If not, use the caption.
-				}
-				if ( empty( $attr['alt'] ) ) {
-					$attr['alt'] = ''; // For a11y reasons, an empty string is better than the attachment name.
-				}
+			// Update image alt tag if not set.
+			if (
```

Comment:
PHP recommended format for multi-line ifs is to precede each line with an operator, thus:

```php
<?php
if ( ! isset( $attr['alt'] )
	&& ! empty( $attachment_id )
	&& is_numeric( $attachment_id )
) {
}
```